tools/sof-dump-status.py

```diff
@@ -2,10 +2,20 @@
 
 import subprocess
 import os
+import sys
+
 from common import format_pipeline, export_pipeline
 
+
+class NoMapException(Exception):
+    """Raise if some element doesn't have an associated map."""
+
 class clsSYSCardInfo():
     def __init__(self):
+        # i.MX-specific maps
+        self.imx_soc_id_to_fw_name = {"i.MX8QM": "imx8"}
+        self.imx_soc_id_to_dsp_name = {"i.MX8QM": "556e8000.dsp"}
```

Comment:
these are constant, make that clearer with capitals at the script's top level.

```suggestion
        IMX_SOC_ID_TO_DSP_NAME = {"i.MX8QM": "556e8000.dsp"}
```

I see you're trying to be consistent with pci_ids below but in this particular case please don't bother. This script has many design issues and these are just CONSTANTS so better do the Right Thing, make them look like constants and make your code more readable. Consistency does matter for complex and/or logical problems, not for defining constants...

Reply:
You're right. I will move them at the beginning of the script and use capitals.